starter-code/index.html

```diff
@@ -2,30 +2,63 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+  <section>
+    <div class="image">
+      <img src="./images/phones.png" alt="phones">
+    </div>
+
+    <div class="content">
+      <div class="content-login">
+        <div class="insta-logo">
+          <img src="./images/brand.png" alt="brand">
+        </div>
+
+        <div class="login">
+          <img src="./images/home.png" alt="login"> 
+          <p>Log in</p>
+        </div>
+      </div>
+
+      <div class="content-text">
+        <div class="border">
+          <h2>Capture and Share <br> the World's Moments</h2>
+          <p>Instagram is a <span>fast</span>, <span>beautiful</span>  and <span>fun</span>  way to share your life with friends and family.</p>
+          <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+          <p>Oh yeah, did we mention it's free?</p>
+          <div class="logo-download">
+            <img src="./images/badge-iphone.png" class="iphone" alt="badge-iphone">
+            <img src="./images/badge-android.png" alt="badge-android">
+        </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <footer>
+    <ul>
+      <li><a href="#0">About Us</a></li>
+      <li><a href="#0">Support</a></li>
+      <li><a href="#0">Blog</a></li>
+      <li><a href="#0">Press</a></li>
+      <li><a href="#0">API</a></li>
+      <li><a href="#0">Jobs</a></li>
+      <li><a href="#0">Privacy</a></li>
+      <li><a href="#0">Terms</a></li>
+      <li><a href="#0" class="copyright">&copy; 2014 Instagram</a></li>
```

Comment:
Nothing to say except well done! flex boxes are well used and your code is clean! 

Hope you had a good weekend ;)